GOVERNANCE.md

```diff
@@ -0,0 +1,135 @@
+### io.js Build Team Working Group
+
+The io.js build team is jointly governed by a Working Group (WG)
+which is responsible for high-level guidance of the project.
+
+The WG has final authority over this project including:
+
+* Technical direction
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
+
+For the current list of WG members, see the project
+[README.md](./README.md#current-project-team-members).
+
+### Collaborators
+
+The [iojs/build](https://github.com/iojs/build),
+[iojs/docker-iojs](https://github.com/iojs/docker-iojs), and the
+[iojs/build-containers](https://github.com/iojs/build-containers)
+GitHub repositories are maintained by the WG and additional Collaborators
+who are added by the WG on an ongoing basis.
+
+Individuals making significant and valuable contributions are made
+Collaborators and given commit-access to the project. These
+individuals are identified by the WG and their addition as
+Collaborators is discussed during the weekly WG meeting.
```

Comment:
That last line was part of the boilerplate before the meeting ever started, but during the call @rvagg specifically said ["not next week"](https://www.youtube.com/watch?v=OKQi3pTF7fs&t=1h2m20s) and then there was no conclusion for what would be a good time. (yay for publicly archived video!)